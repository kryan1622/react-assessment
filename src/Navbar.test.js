import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Example from './Components/navbar.js';


it('render navbar', () => {
  const wrapper = renderer.create(<Example/>).toJSON()
  expect(wrapper).toMatchSnapshot();
})