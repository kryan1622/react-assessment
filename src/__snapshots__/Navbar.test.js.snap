// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render navbar 1`] = `
<div>
  <nav
    className="navbar navbar-expand-md navbar-dark bg-dark"
  >
    <a
      aria-current="page"
      className="navbar-brand active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      User Login Site
    </a>
    <ul
      className="ml-auto navbar-nav"
    >
      <li
        className="nav-item"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-users fa-w-20 "
          data-icon="users"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 640 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M96 224c35.3 0 64-28.7 64-64s-28.7-64-64-64-64 28.7-64 64 28.7 64 64 64zm448 0c35.3 0 64-28.7 64-64s-28.7-64-64-64-64 28.7-64 64 28.7 64 64 64zm32 32h-64c-17.6 0-33.5 7.1-45.1 18.6 40.3 22.1 68.9 62 75.1 109.4h66c17.7 0 32-14.3 32-32v-32c0-35.3-28.7-64-64-64zm-256 0c61.9 0 112-50.1 112-112S381.9 32 320 32 208 82.1 208 144s50.1 112 112 112zm76.8 32h-8.3c-20.8 10-43.9 16-68.5 16s-47.6-6-68.5-16h-8.3C179.6 288 128 339.6 128 403.2V432c0 26.5 21.5 48 48 48h288c26.5 0 48-21.5 48-48v-28.8c0-63.6-51.6-115.2-115.2-115.2zm-223.7-13.4C161.5 263.1 145.6 256 128 256H64c-35.3 0-64 28.7-64 64v32c0 17.7 14.3 32 32 32h65.9c6.3-47.4 34.9-87.3 75.2-109.4z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <a
          href="/find"
          onClick={[Function]}
          style={
            Object {
              "color": "#87898C",
            }
          }
          tag={[Function]}
        >
          Find Users
        </a>
      </li>
      <li
        className="nav-item"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-user-edit fa-w-20 "
          data-icon="user-edit"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 640 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h274.9c-2.4-6.8-3.4-14-2.6-21.3l6.8-60.9 1.2-11.1 7.9-7.9 77.3-77.3c-24.5-27.7-60-45.5-99.9-45.5zm45.3 145.3l-6.8 61c-1.1 10.2 7.5 18.8 17.6 17.6l60.9-6.8 137.9-137.9-71.7-71.7-137.9 137.8zM633 268.9L595.1 231c-9.3-9.3-24.5-9.3-33.8 0l-37.8 37.8-4.1 4.1 71.8 71.7 41.8-41.8c9.3-9.4 9.3-24.5 0-33.9z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <a
          href="/register"
          onClick={[Function]}
          style={
            Object {
              "color": "#87898C",
            }
          }
          tag={[Function]}
        >
          Register
        </a>
      </li>
      <li
        className="nav-item"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-user-circle fa-w-16 "
          data-icon="user-circle"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 496 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M248 8C111 8 0 119 0 256s111 248 248 248 248-111 248-248S385 8 248 8zm0 96c48.6 0 88 39.4 88 88s-39.4 88-88 88-88-39.4-88-88 39.4-88 88-88zm0 344c-58.7 0-111.3-26.6-146.5-68.2 18.8-35.4 55.6-59.8 98.5-59.8 2.4 0 4.8.4 7.1 1.1 13 4.2 26.6 6.9 40.9 6.9 14.3 0 28-2.7 40.9-6.9 2.3-.7 4.7-1.1 7.1-1.1 42.9 0 79.7 24.4 98.5 59.8C359.3 421.4 306.7 448 248 448z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <a
          href="/"
          onClick={[Function]}
          style={
            Object {
              "color": "#87898C",
            }
          }
          tag={[Function]}
        >
          Login
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="container"
  >
    <br />
    <br />
    <h4>
       Login 
    </h4>
    <br />
    <br />
    <form
      onSubmit={[Function]}
    >
      <label
        className=""
      >
        Username
      </label>
      <input
        className="form-control"
        type="text"
      />
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="Password"
        >
          Password
        </label>
        <input
          className="form-control"
          type="password"
        />
      </div>
      <br />
      <p
        style={
          Object {
            "color": "red",
          }
        }
      >
        
      </p>
      <input
        name="clickme"
        type="submit"
        value="Submit"
      />
    </form>
  </div>
</div>
`;
